### PR TITLE
[ES] Add `Beatmap/Hype`

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by DoryanR** [comment]
The file you've translated is `en.md`, not `es.md`. Moreover, it contains the same changes from #10317

**Comment by agatemosu** [comment]
thanks

**Comment by agatemosu** [comment]
> Moreover, it contains the same changes from #10317

because this branch is based on a commit of that PR

File: wiki/Beatmap/Hype/es.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+stub: true
+tags:
+  - train
+  - tren
+---
+
+# Hype
+
+Cada [beatmap](/wiki/Beatmap) [pendiente](/wiki/Beatmap/Category#trabajos-en-progreso-y-pendientes) tiene un **tren del hype** que representa aproximadamente cuántas personas están interesadas en ver el mapa [clasificado](/wiki/Beatmap/Category#clasificados). Los usuarios pueden publicar [discusiones de beatmaps](/wiki/Beatmap_discussion) especiales en la pestaña `General (All difficulties)` con la opción `Hype!` para contribuir al tren del hype de un mapa.
```
Comment (UnknownOsu) [inline wiki/Beatmap/Hype/es.md:10]:
```suggestion
Cada [beatmap](/wiki/Beatmap) [pendiente](/wiki/Beatmap/Category#trabajos-en-progreso-y-pendientes) tiene un **tren del hype** que representa aproximadamente cuántas personas están interesadas en ver el mapa [clasificado](/wiki/Beatmap/Category#clasificados). Los usuarios pueden publicar [discusiones de beatmaps](/wiki/Beatmap_discussion) especiales en la pestaña `General (Todas las dificultades)` con la opción `¡Hype!` para contribuir al tren del hype de un mapa.
```

File: wiki/Beatmap/Hype/es.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## Requisito de clasificación
+
+Los beatmaps necesitan al menos 5 hypes (una barra completa del tren de hype) para ser elegibles para la [nominación](/wiki/Beatmap_ranking_procedure#nominations), un paso para clasificar el mapa.
```
Comment (UnknownOsu) [inline wiki/Beatmap/Hype/es.md:14]:
```suggestion
Los beatmaps necesitan al menos 5 hypes (una barra completa del tren del hype) para ser elegibles para la [nominación](/wiki/Beatmap_ranking_procedure#nominations), un paso para clasificar el mapa.
```

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

**Review by UnknownOsu — APPROVED** [review]

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**